content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md

```diff
@@ -0,0 +1,135 @@
+This is where things become hard. Huawei seldom provide detailed statistics of sales volume for it's models. Bits and pieces are sometimes dropped through annual reports or an occasional press release. So let's start with some known facts.
+
+### P8 Lite
+We know that by Februar 2016, the P8 Lite had shipped above 10 million units, since it's release in April 2015. The entire P8 series (all P8 models) had by the same time shipped above 16 million units [(source)](http://consumer.huawei.com/en/press/news/2016/hw-472293/). Huawei have provided a detailed monthly shipment chart for this specific model, which is quite unique, and therefore an important data source.
+
+![P8 Lite shipment trend](http://img.c.huaweistatic.com/content/dam/huawei-cbg-site/en/mkt/press/news/2016/31609-1.jpg "P8 Lite shipment trend")
+
+### P9
+Although the P8 Lite managed to sell above 10 million units, P9 was credited by Huawei as beeing the first _"flagship"_ device to do so.
+
+### General sales
+Here is a table trying to give every known sales volumes by offical (or at least credible) sources, for all Huawei and Honor models that might be relevant for predicting the sales volumes of the models running Vidhance. _Italic data points_ are estimations based upon other __known datapoints, marked in bold__. Paranthesis mark the source of information for the datapoint.
+
+| Model | Released | Month 1 | Month 2 | Month 3  | Month 4  | Month 5  | Month 6  | Month 7  | Month 8  | Month 9  | Month 10  | Month 11  | Month 12|
+|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+| **P7** | 2014 May | - | - | - | - | - | - | - | - | - | - | - | **6M** [(1)][P7_12MONTH] |
+| **P8 Lite** | 2015 April | _500k_ | _1.4M_ | _2.2M_ | _3.2M_ | _4.38M_ | _5.58M_ | _7.08M_ | _8.58M_ | **10M** [(1)][P8LITE_9MONTH] | - | - | - |
+| **P8** | 2015 April | - | - | - | - | - | - | - | - | - | - | _4.77M_ | - |
+| **P9** | 2016 April | - | **>2.6M** [(1)][P9_2MONTH] | - | - | **6M** [(1)][P9_5MONTH] | - |- |  **9M** [(1)][P9_9MONTH] | - | - | **12M** [(1)][P9_12MONTH] | - |
+| **P10** | 2017 Mar | - | - | **6M** [(1)][P10_3MONTH] | - | - | - | - | - | - | - | - | - |
+| **Mate 8** | 2015 Dec | **>1M** [(1)][M8_1MONTH] | - | - | _3.68M_ | - | - | - | - | - | - | - | 7M [(1)][M8_12MONTH] |
+| **Mate 9** | 2016 Dec | - | - | - | **5M** [(1)][M9_4MONTH] | - | - | **8.5M** [(1)][M9_4MONTH] | - | - | - | - | - |
+| **Honor 8** | 2016 Jul | - | **1.5M** [(1)][H8_2MONTH] | - | - | - | - | - | - | - | - | - | - |
+| **Honor 9** | 2017 Jun | **>1M** [(1)][H9_1MONTH] | - | - | - | - | - | - | - | - | - | - | - |
+
+### Mate 9 Family
+The Mate 9 family includes the "Mate 9", "Mate 9 Pro" and "Mate 9 Porsche design". The "Mate 9 Lite" is not included, as it does not run Vidhance software. The Mate 9 Family was announced in October 2016, and went on sale in China in November. Availability in most of western Europa, along with the US was not in place until January 2017.
```

Comment:
Maybe state very early in the article that Lite are totally different models and then define the Mate 9 Series and use that everywhere.